src/Common/src/Logging/UpgradableBootstrapLoggerFactory.cs

```diff
@@ -122,31 +121,37 @@ private static void DefaultConfigure(ILoggingBuilder builder, IConfiguration con
         builder.AddConsole().AddConfiguration(configuration.GetSection("Logging"));
     }
 
-    internal sealed class BootstrapLoggerInst : ILogger
+    private sealed class BootstrapLoggerInstance : ILogger
     {
-        public volatile ILogger Logger;
+        private volatile ILogger _logger;
 
         public string Name { get; }
 
-        public BootstrapLoggerInst(ILogger logger, string name)
+        public BootstrapLoggerInstance(ILogger logger, string name)
```

Comment:
I know, there are more locations this applies to, as well as other optimizations. But I've held back taking Resharper suggestions to avoid making this PR longer. Reviewing them is on my list of pending work, they will be included in a follow-up PR.